packages/web-config-server/src/apiV1/DataAggregatingRouteHandler.js

```diff
@@ -20,36 +20,27 @@ export class DataAggregatingRouteHandler extends RouteHandler {
 
   // Builds the list of entities data should be fetched from, using org unit descendants of the
   // selected entity (optionally of a specific entity type)
-  fetchDataSourceEntities = async (entity, dataSourceEntityType, dataSourceEntityFilter) => {
+  fetchDataSourceEntities = async (entity, options) => {
+    const { dataSourceEntityType } = options;
```

Comment:
In general, do you prefer 'stripping out' unneeded config options? e.g.

```suggestion
    const { dataSourceEntityType, ...restOfOptions } = options;
```

and then passing in `restOfOptions` and not `options`?

Reply:
I think I'd prefer `restOfOptions` . Maybe it's just coding style but I prefer passing only the parameters you need...